package.json

```diff
@@ -44,7 +44,7 @@
     "react-dom": "^0.14.9 || >=15.3.0"
   },
   "devDependencies": {
-    "@monastic.panic/component-playground": "^2.0.0",
+    "@monastic.panic/component-playground": "^3.2.2",
```

Comment:
Wouldn't it be better to do that upgrade in a separate PR? Would be easier to review IMO.

Reply:
I updated this to verify stuff was working apart from the tests which are flaky for me. I think it should be ok